geopandas/tools/tests/test_geocode.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
```

Comment:
Can you move these to existing `geopandas/tests/test_geocode.py`?

Reply:
Sure thing :)